internal/saucecloud/espresso.go

```diff
@@ -163,17 +163,13 @@ func (r *EspressoRunner) startJob(jobOpts chan<- job.StartOptions, s espresso.Su
 		jto.UseTestOrchestrator = &s.TestOptions.UseTestOrchestrator
 	}
 
-	for i, ID := range otherAppsIDs {
-		otherAppsIDs[i] = fmt.Sprintf("storage:%s", ID)
-	}
-
 	jobOpts <- job.StartOptions{
 		DisplayName:       displayName,
 		Timeout:           s.Timeout,
 		ConfigFilePath:    r.Project.ConfigFilePath,
 		CLIFlags:          r.Project.CLIFlags,
-		App:               fmt.Sprintf("storage:%s", appFileID),
-		Suite:             fmt.Sprintf("storage:%s", testAppFileID),
+		App:               appFileID,
+		Suite:             testAppFileID,
```

Comment:
A tiny bit conflicted about this one, due to the resulting, now slightly misleading, variable name 😅 

`123-456-abc` is an ID.
With this change, you are referring to `storage:123-456-abc` as an ID. But it's actually a URI now.

Reply:
I will update the name, as it may be really misleading in a couple of week/month.